3-With-Intune MAM/README.md

```diff
@@ -0,0 +1,44 @@
+# Integrate Microsoft identity, the Microsoft Authenticator App (broker), and Microsoft Intune MAM SDK into a Xamarin forms app using MSAL
+
+This sample demonstrates how to access Intune MAM protected resources using MSAL.NET
+
+## Overview
+When a user attempts to access a MAM protected resource and if the device is not compliant, MSAL.NET will throw ```IntuneAppProtectionPolicyRequiredException```. The common code catches the exception and requests Intune MAM SDK to make it compliant. These requests have platform specific implementation. So they are executed using IoC pattern i.e. using an interface with platform specific implementation.  
+
+After the device becomes compliant, the App makes silent token request to obtain the token.  
+
+**Note:** The sample accounts only for the scenario when user completes compliance request. Handling of scenarios where uesr does not satisfy the compliance request will be up to the business requirements of the application.
+
+## How To Run this Sample

```

Comment:
Why not add instructions on how to use the `.tour` featuere here? Just a thought, that it's a nice feature.